test/DI.Tests/ServiceCollectionServiceExtensionsTest.cs

```diff
@@ -211,11 +211,14 @@ public static TheoryData TryAddImplementationTypeData
             ServiceLifetime expectedLifetime)
         {
             // Arrange
-            var collection = new ServiceCollection();
-            collection.Add(ServiceDescriptor.Transient(expectedServiceType, expectedServiceType));
+            var collection = new ServiceCollection
+            {
+                ServiceDescriptor.Transient(expectedServiceType, expectedServiceType)
+            };
 
             // Act
             addAction(collection);
+            collection.TryAdd(ServiceDescriptor.Describe(expectedServiceType, expectedImplementationType, expectedLifetime));
```

Comment:
I think it fits with the description of the test: "TryAdd_WithType_DoesNotAddDuplicate". But you're welcome to refactor this test if it bothers you enough.